### ISSUE TITLE
Update CMS after BetterButtonNestedForm submit

### ISSUE THREAD

**Issue by sanderha** [unclecheese/silverstripe-gridfield-betterbuttons#126]
Hello

Im using ```BetterButtonNestedForm``` which works perfectly. The only problem is it does not update the information shown in the CMS after submitting the form.

When you use a normal ``` BetterButtonCustomAction ```, it will update the view for you, with any updated information changed by this action. The ``` BetterButtonNestedForm``` does not seem to do this though.

Not the biggest issue, but thought I'd let you know. Great module all in all.

**Comment by unclecheese** [comment]
Yeah, I'm aware of this. Not an easy thing to solve, and unfortunately, it has huge impacts on usabilty. :(

**Comment by sanderha** [comment]
Damn.. There isnt any way to force refresh everything? ;)

**Comment by unclecheese** [comment]
I'm sure there is. I know there's an `onClose` hook for that modal window. We could do something with that, but then you end up refreshing regardless of what the user did or did not do in the nested form. 

It just needs some thought, and I haven't really had a chance to look at it much. This is one of those power-user features that doesn't get a lot of attention or testing, but I'm keen to get it right.
